convert2rhel/checks.py

```diff
@@ -46,6 +46,18 @@
 KERNEL_REPO_VER_SPLIT_RE = re.compile(r"\W+")
 BAD_KERNEL_RELEASE_SUBSTRINGS = ("uek", "rt", "linode")
 
+RPM_GPG_KEY_PATH = os.path.join(utils.DATA_DIR, "gpg-keys", "RPM-GPG-KEY-redhat-release")
+# The SSL certificate of the https://cdn.redhat.com/ server
+SSL_CERT_PATH = os.path.join(utils.DATA_DIR, "redhat-uep.pem")
+REPO_CONTENT = (
+    "[convert2rhel]\n"
+    "name=Convert2RHEL Repository\n"
+    "baseurl=https://cdn.redhat.com/content/public/convert2rhel/$releasever/$basearch/os/\n"
+    "gpgcheck=1\n"
+    "enabled=1\n"
+    "sslcacert=%s\n"
+    "gpgkey=file://%s\n" % (SSL_CERT_PATH, RPM_GPG_KEY_PATH)
+)
```

Comment:
I would suggest using the triple quotes (`"""`) for this to make the readability better, for example: 

```suggestion
REPO_CONTENT = """\
[convert2rhel]
name=Convert2RHEL Repository
baseurl=https://cdn.redhat.com/content/public/convert2rhel/$releasever/$basearch/os/
gpgcheck=1
enabled=1
sslcacert=%s
gpgkey=file://%s
""" % (SSL_CERT_PATH, RPM_GPG_KEY_PATH)
```

Not something that you need to add only if you think that it improves the readability of the code,

Reply:
I have implemented this change in the code. thank you for the advice!